src/app/multi-editor/multi-editor.component.ts

```diff
@@ -264,6 +251,40 @@ export class MultiEditorComponent implements OnInit {
     this.changeDetectorRef.markForCheck();
   }
 
+  private checkExistenceOfAction(userActions: UserActions): boolean {
+    let existingAction = false;
+    return userActions.actions.some((element, index, array) => {
```

Comment:
would be nice to have a new line with `.some`

Reply:
only first param is enough here, so with a renaming this would be `.some(action => {..`